dialog/file.go

```diff
@@ -476,21 +476,23 @@ func (f *FileDialog) Resize(size fyne.Size) {
 	if f.dialog == nil {
 		return
 	}
-	maxSize := f.dialog.win.Size()
 	minSize := f.dialog.win.MinSize()
-	newWidth := size.Width
-	if size.Width > maxSize.Width {
-		newWidth = maxSize.Width
-	} else if size.Width < minSize.Width {
-		newWidth = minSize.Width
-	}
-	newHeight := size.Height
-	if size.Height > maxSize.Height {
-		newHeight = maxSize.Height
-	} else if size.Height < minSize.Height {
-		newHeight = minSize.Height
-	}
-	f.dialog.win.Resize(fyne.NewSize(newWidth, newHeight))
+	f.dialog.win.Resize(minSize.Max(size))
+	// maxSize := f.dialog.win.Size()
```

Comment:
Please don't commit commented out code.

Additionally I don't think that removing the checks that the dialog fits within the window bounds is the right fix here.
You'll see that with this the file dialog no longer shrinks down to minsize if the window is smaller than requested size.

Reply:
> Please don't commit commented out code.

Sorry about it.

> Additionally I don't think that removing the checks that the dialog fits within the window bounds is the right fix here.
You'll see that with this the file dialog no longer shrinks down to minsize if the window is smaller than requested size.

They are fixed with #1931.